torch/linalg/__init__.py

```diff
@@ -92,11 +91,22 @@
 """)
 
 inv = _add_docstr(_linalg.linalg_inv, r"""
-linalg.inv(input, *, out=None) -> Tensor
+linalg.inv(A, *, out=None) -> Tensor
+
+Computes the `inverse`_ of square matrix if it exists.
+
+Letting :math:`\mathbb{K}` be :math:`\mathbb{R}` or :math:`\mathbb{C}`,
+for a matrix :math:`A \in \mathbb{K}^{n \times n}`,
+its inverse matrix :math:`A^{-1} \in \mathbb{K}^{n \times n}` is defined as
```

Comment:
should this add a parenthetical to explain that the inverse may not exist?

"it inverse matrix :math:..., if it exists, is defined as" ?

Reply:
Alternatively this could be reworded to explain that an inverse for a matrix is a matrix such that... and then it could say that if a matrix has an inverse it's "invertible" and the inverse is unique, but that not every matrix has an inverse, and those matrices are described as "not invertible."

What would you think about that approach?

Reply:
I'll do both. Put first the parenthesis (if it exists) and then describe in the next paragraph when may it not exist. Same for `torch.linalg.eig`.